python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -1035,15 +1036,15 @@ async def move_camera_hexapod(
             Hexapod-v angle (degrees).
         w : `float`, optional
             Hexapod-w angle (degrees). Default 0.
-        sync : `bool`, optinal
+        sync : `bool`, optional
             Should the hexapod movement be synchronized? Default True.
         """
 
         compensation_mode = await self.get_compensation_mode_camera_hexapod()
 
         if compensation_mode.enabled:
             self.log.info(
-                "Camera Hexapod compensation mode enabled. Move will offset with respect to LUT."
+                "Camera Hexapod compensation mode enabled. Move with respect to LUT."
```

Comment:
Do we want to add an `else` here with an info message saying something like: 
```Camera Hexapod compensation mode disabled. Move to absolute position.```?